node/node.go

```diff
@@ -91,6 +91,7 @@ type nodeChainReader interface {
 	GetTipSetStateRoot(tsKey types.SortedCidSet) (cid.Cid, error)
 	HeadEvents() *ps.PubSub
 	Load(context.Context) error
+	SetHead(context.Context, types.TipSet) error
```

Comment:
We still need to figure out a way to get rid of this write method. Is this still necessary?

Reply:
Blocking: same comment as before, no node caller should ever be updating chain store state from node.  (Sorry if it wasn't clear that this applied to SetHead as well as PutTipSetAndState)